airbyte-integrations/connectors/source-klaus-api/source_klaus_api/schemas/categories.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {"type": ["string"]},
```

Comment:
Can you set additionalProperties to true for all of the schema files? This will help ensure that future api changes will not break the connector so easily

Reply:
additionalProperties also need to be set for each of the schema files, the property in the spec.yaml file is separate. Also in our docs although not as verbose: https://docs.airbyte.com/connector-development/schema-reference/